src/Tools/Source/RunTests/Program.cs

```diff
@@ -261,19 +261,19 @@ async Task DumpProcess(Process targetProcess, string procDumpExeFilePath, string
                 var output = await ProcessRunner.CreateProcess("Powershell.exe", $"-command \"& {{ . .\\eng\\build-utils-win.ps1; Capture-Screenshot {screenshotPath} }}\"", displayWindow: false, cancellationToken: cancellationToken).Result;
                 ConsoleUtil.WriteLine(string.Join(Environment.NewLine, output.OutputLines));
                 ConsoleUtil.WriteLine(string.Join(Environment.NewLine, output.ErrorLines));
-            }
-
-            if (options.CollectDumps && !string.IsNullOrEmpty(options.ProcDumpFilePath))
-            {
-                ConsoleUtil.WriteLine("Roslyn Error: test timeout exceeded, dumping remaining processes");
 
-                var counter = 0;
-                foreach (var proc in ProcessUtil.GetProcessTree(Process.GetCurrentProcess()).OrderBy(x => x.ProcessName))
+                if (options.CollectDumps && !string.IsNullOrEmpty(options.ProcDumpFilePath))
                 {
-                    var dumpDir = options.LogFilesDirectory;
-                    var dumpFilePath = Path.Combine(dumpDir, $"{proc.ProcessName}-{counter}.dmp");
-                    await DumpProcess(proc, options.ProcDumpFilePath, dumpFilePath);
-                    counter++;
+                    ConsoleUtil.WriteLine("Roslyn Error: test timeout exceeded, dumping remaining processes");
```

Comment:
Why do we have to change this tool?

Reply:
This up gating the GetProcessTree code, code that will only work on Windows, under the if above: `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))`, to resolve compiler errors around unsupported platform use.